MinecraftClient/Mapping/Vector3.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MinecraftClient.Mapping
+{
+    /// <summary>
+    /// Simple vector3 class
+    /// </summary>
+    public class Vector3
```

Comment:
This class looks like a duplicate of [Location](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/Mapping/Location.cs) which already has the Add and Multiply properties with `+` and `*` operators.